website/docs/docs/build/metricflow-time-spine.md

```diff
@@ -463,9 +463,22 @@ For example, if you use a custom calendar in your organization, such as a fiscal
 
 - This is useful for calculating metrics based on a custom calendar, such as fiscal quarters or weeks. 
 - Use the `custom_granularities` key to define a non-standard time period for querying data, such as a `retail_month` or `fiscal_week`, instead of standard options like `day`, `month`, or `year`.
-- Ensure the the `standard_granularity_column` is a date time type.
 - This feature provides more control over how time-based metrics are calculated.
 
+<Expandable alt_header="Data types and time zone considerations">
+ 
+When working with custom calendars in MetricFlow, it's important to ensure:
+
+- Consistent data types &mdash; Both your dimension column and the time spine column should use the same data type to allow accurate comparisons. Functions like `DATE_TRUNC` don't change the data type of the input in some databases (like Snowflake). Using different data types can lead to mismatches and inaccurate results.
+
+  We recommend using `DATETIME` or `TIMESTAMP` data types for your time dimensions and time spine, as they support all granularities. The `DATE` data type may not support higher granularities like hours or minutes.
+
+- Consistent time zones &mdash; Ensure that all your time-related data uses the same time zone. MetricFlow supports UTC and currently doesn't perform any timezone manipulation. This means inconsistent time zones can cause unexpected results during aggregations and comparisons.
```

Comment:
I would change this section slightly:
`Time zones &mdash; MetricFlow currently doesn't perform any timezone manipulation. When working with timezone-aware data, inconsistent time zones may lead to unexpected results during aggregations and comparisons.`
Reasons for the change:
- MF doesn't support UTC per se, it just ignores timezones entirely. UTC is just the most common way that people are likely to store their data for timezone consistency.
- I don't think we should ask them to ensure all their data uses the same timezone since they might have a valid reason for not doing that. We just want to warn them that there might be weird interactions in that case.

Reply:
good callout!